Problems/5585/5585_ans_lhm.py

```diff
@@ -0,0 +1,28 @@
+#https://www.acmicpc.net/problem/5585
+import sys
+input = sys.stdin.readline
+
+N = int(input())
+cal_Result = 1000-N
+cnt = 0
+while cal_Result>0:
+    if cal_Result>=500:
+        cal_Result-=500
+        cnt+=1
```

Comment:
문제 풀이는 어렵지 않은데, 이렇게 되면 연산되는 횟수가 많지 않을까?

다음처럼 나누기를 한 몫을 이용하면 while 반복문 횟수가 많이 줄지 않을까 싶어요

```suggestion
    if cal_Result // 500 > 0:
        cal_Result-=500 * (cal_Result//500)
        cnt+=cal_Result//500
```